client/me/developer/features/use-features-list.tsx

```diff
@@ -0,0 +1,104 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+
+export const useFeaturesList = () => {
+	const translate = useTranslate();
+
+	return [
+		{
+			title: translate( 'SFTP, SSH, and WP-CLI', {
+				comment: 'Feature title',
+			} ),
+			description: translate(
+				'Run WP-CLI commands, automate repetitive tasks, and troubleshoot your custom code with the tools you already use.',
+				{
+					comment: 'Feature description',
+				}
+			),
+			linkLearnMore: localizeUrl( '/support/connect-to-ssh-on-wordpress-com' ),
+		},
+		{
+			title: translate( 'Staging sites', {
+				comment: 'Feature title',
+			} ),
+			description: translate(
+				'Test changes on a WordPress.com staging site first, so you can identify and fix any vulnerabilities before they impact your live site.',
+				{
+					comment: 'Feature description',
+				}
+			),
+			linkLearnMore: localizeUrl( '/support/how-to-create-a-staging-site/' ),
+		},
+		{
+			title: translate( 'Custom code', {
+				comment: 'Feature title',
+			} ),
+			description: translate(
+				'Build anything with support and automatic updates for 50,000+ plugins and themes. Or start from scratch with your own custom code.',
+				{
+					comment: 'Feature description',
+				}
+			),
+			linkLearnMore: localizeUrl( '/support/code' ),
+		},
+		{
+			title: translate( 'Free SSL certificates', {
+				comment: 'Feature title',
+			} ),
+			description: translate(
+				'Take your site from HTTP to HTTPS at no additional cost. We encrypt every domain registered and connected to WordPress.com with a free SSL certificate.',
+				{
+					comment: 'Feature description',
+				}
+			),
+			linkLearnMore: localizeUrl( '/support/domains/https-ssl' ),
+		},
+		{
+			title: translate( '24/7 expert support', {
+				comment: 'Feature title',
+			} ),
+			description: translate(
+				"Whenever you're stuck, whatever you're trying to make happen—our Happiness Engineers have the answers.",
```

Comment:
It was done intentionally, I also didn't know, but it turned out that avoiding spaces is more right way. But according to Google - there is no right way, both options are used by people :)

Reply:
Here's an example we have no spaces around the em-dash: https://github.com/Automattic/wp-calypso/blob/fabdd100c4406dfd8dd350b473165c82142ecb8a/client/my-sites/domains/domain-management/list/google-domain-owner-banner.tsx#L19-L21

It would be nice to be consistent in all of Calypso, but by changing the copies now, the updated phrases will need to be re-translated again.